🌐 [i18n-KO] Translated `perf_train_gpu_one.md` to Korean

D15M4S opened the pull request:
<!-- PR의 제목은 "🌐 [i18n-KO] Translated `<your_file>.md` to Korean" 으로 부탁드립니다 -->
# What does this PR do?

Translated the `perf_train_gpu_one.md` file of the documentation to Korean.
Thank you in advance for your review.

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [x] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

<!-- 1. 위 체크가 모두 완료된 뒤에만 KREW 팀원들에게 리뷰를 요청하는 아래 주석을 노출해주세요!-->
May you please review this PR?
@4N3MONE, @yijun-lee, @jungnerd , @harheem 

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

<!-- 2. KREW 팀원들의 리뷰가 끝난 후에 아래 주석을 노출해주세요! -->
@stevhliu May you please review this PR?

D15M4S committed:
docs: ko: perf_train_gpu_one.md

D15M4S committed:
feat: nmt draft

D15M4S committed:
fix: manual edits

D15M4S committed:
fix: Manually added missing backticks

harheem commented:
`Translated the <your_file>.md file of the documentation to Korean.`
=> PR 본문의 perf_train_gpu_one.md 로 수정 부탁드려요!

D15M4S commented:
> `Translated the <your_file>.md file of the documentation to Korean.` => PR 본문의 perf_train_gpu_one.md 로 수정 부탁드려요!

수정 완료했습니다!

stevhliu reviewed (approved):
Thanks, feel free to mark as ready once you're ready! 🤗

4N3MONE commented on docs/source/ko/perf_train_gpu_one.md:
```diff
@@ -0,0 +1,294 @@
+
+아래 표를 참고하면 자신의 학습 시나리오에 적합한 기능을 빠르게 파악할 수 있습니다.
+
+| 기능                        | 학습 속도 | 메모리 사용량 |
```
```suggestion
| 기능                        | 학습 속도 가속 | 메모리 사용량 절약 |
```

4N3MONE reviewed (commented):
번역 고생하셨습니다! 🤗

D15M4S committed:
Update docs/source/ko/perf_train_gpu_one.md

fix: remove space between heading and GPU anchor

Co-authored-by: YONGSANG <71686691+4N3MONE@users.noreply.github.com>

D15M4S committed:
Update docs/source/ko/perf_train_gpu_one.md

fix: clarify table headers to indicate training speed boost and memory savings

Co-authored-by: YONGSANG <71686691+4N3MONE@users.noreply.github.com>

D15M4S commented:
> 번역 고생하셨습니다! 🤗

감사합니다! 의견 주신 내용 모두 반영했습니다! 😄

harheem reviewed (commented):
타이틀 부분 제안드려봅니다 :)
문서 번역하시느라 고생하셨고, 감사합니다 ☺️

harheem commented on docs/source/ko/_toctree.yml:
```diff
@@ -176,8 +176,8 @@
-    - local: in_translation
-      title: (번역중) Methods and tools for efficient training on a single GPU
+    - local: perf_train_gpu_one
+      title: GPU
```
앗 그렇군요! 감사합니다 GPU로 그대로 가면될 것 같습니다~~~

harheem reviewed (approved):
LGTM!
매뉴얼 수정도 잘 진행해주셨고 전반적으로 문서가 정말 잘 읽히네요 고생하셨고, 감사합니다 ☺️

jungnerd commented on docs/source/ko/perf_train_gpu_one.md:
```diff
@@ -0,0 +1,294 @@
+
+GPU는 높은 메모리 대역폭과 병렬 처리 능력 덕분에 딥러닝 모델 학습에 널리 사용됩니다. GPU 사양과 모델 크기에 따라 수십억 개 매개변수를 가진 모델도 학습할 수 있습니다. 핵심은 GPU 메모리 활용도(데이터 처리량/학습 시간)와 학습 속도 사이에서 최적의 균형을 찾는 것입니다.
+
+이 가이드는 Transformers와 PyTorch에서 GPU에서 모델을 효율적으로 학습하기 위해 제공하는 기능을 소개합니다. 대부분 이러한 기능을 조합하여 학습을 최적화하는 것이 좋습니다.
```
```suggestion
이 가이드는 Transformers와 PyTorch에서 GPU를 활용해 모델을 효율적으로 학습하기 위해 제공하는 기능을 소개합니다. 대부분의 경우, 이 기능들을 조합해서 학습을 최적화하는 것이 좋습니다.
```
'에서'라는 조사가 연속으로 나오는 게 어색해 수정했습니다. 그다음 문장도 다르게 번역해봤습니다.

jungnerd commented on docs/source/ko/perf_train_gpu_one.md:
```diff
@@ -0,0 +1,294 @@
+
+### 데이터 사전 적재[[data-preloading]]
+
+데이터 사전 적재는 GPU가 지속적으로 작업할 수 있도록 CPU에서 미리 배치 단위의 데이터를 적재하고 준비하는 기능입니다. 이를 통해 GPU 유휴 시간을 줄이고 활용도를 높일 수 있습니다. GPU가 항상 작업을 계속하도록 하려면 다음 데이터 사전 적재를 위한 두 가지 방법을 사용할 수 있습니다.
```
```suggestion
데이터 사전 적재(Data preloading)는 GPU가 지속적으로 작업할 수 있도록 CPU에서 미리 배치 단위의 데이터를 적재하고 준비하는 기능입니다. 이를 통해 GPU 유휴 시간을 줄이고 활용도를 높일 수 있습니다. GPU가 항상 작업을 계속하도록 하려면 다음 데이터 사전 적재를 위한 두 가지 방법을 사용할 수 있습니다.
```
'데이터 사전 적재'라는 표현이 낯설 수 있을 것 같아 영문을 병기했습니다.

jungnerd reviewed (commented):
좋은 번역 감사합니다! 몇 가지 수정 사항 제안드려봅니다 😊

D15M4S committed:
Update docs/source/ko/perf_train_gpu_one.md

fix: improve readability

Co-authored-by: Woojun Jung <46880056+jungnerd@users.noreply.github.com>

D15M4S committed:
Update docs/source/ko/perf_train_gpu_one.md

fix : rephrase explanation of data preloading to improve readability

Co-authored-by: Woojun Jung <46880056+jungnerd@users.noreply.github.com>

D15M4S committed:
Merge branch 'main' into ko-perf_train_gpu_one.md

HuggingFaceDocBuilderDev commented:
The docs for this PR live [here](https://moon-ci-docs.huggingface.co/docs/transformers/pr_39552). All of your documentation changes will be reflected on that endpoint. The docs are available until 30 days after the last update.
